platform/device/internal/deviceproto/device.proto

```diff
@@ -33,5 +33,5 @@ message Device {
     string dep_profile_assigned_by =27;
     int64 last_seen =28;
     bytes last_query_response =29;
-
+    bytes device_cert = 30;
```

Comment:
IMO the device cert lives with the CA not the device record.

Reply:
We need to have an association from device record (UDID) to certificate, however. We probably don't need to store the whole cert. I'll likely refactor to just storing a hash of the certificate per the PR note.